app/src/main/java/com/teamagam/gimelgimel/app/control/receivers/GpsStatusBroadcastReceiver.java

```diff
@@ -0,0 +1,77 @@
+package com.teamagam.gimelgimel.app.control.receivers;
+
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+import android.support.v4.content.LocalBroadcastManager;
+
+import com.teamagam.gimelgimel.app.common.ConsistentStatusEventRaiser;
+
+/**
+ * Created by CV on 6/27/2016.
+ */
+public class GpsStatusBroadcastReceiver extends BroadcastReceiver implements ConsistentStatusEventRaiser.EventAction<Boolean> {
+
+    /**
+     * An intent action name, that the receiver should register to
+     */
+    public static final String BROADCAST_GPS_STATUS_ACTION = "newGpsStatus";
+
+    /**
+     * An intent action name, that the BL class should register to
+     */
+    public static final String BROADCAST_NEW_GPS_STATUS_ACTION = "broadCastNewGpsStatus";
+
+    /**
+     * The extra string key
+     */
+    public static final String GPS_STATUS_EXTRA = "gpsStatusExtra";
+
+    public static void broadcastGpsStatus(Context context, boolean gpsStatus) {
+        Intent intent = new Intent(BROADCAST_GPS_STATUS_ACTION);
+        intent.putExtra(GPS_STATUS_EXTRA, gpsStatus);
+
+        LocalBroadcastManager.getInstance(context).sendBroadcast(intent);
+    }
+
+    private Context mApplicationContext;
```

Comment:
We agreed on this order: static - public, private, public methods, private methods. non-static - public, private, constructor, public methods, private methods